pom.xml

```diff
@@ -1405,6 +1405,12 @@
                 <version>2.3</version>
             </dependency>
 
+            <dependency>
```

Comment:
Why is this being added to the parent pom?

Reply:
To have a uniform version all across the project not just module, we can also add the dependencyManagement block in module `presto-product-tests` but then if other modules pull commons-lang3 as transitive dependency with different version, it would be rework one has to do for those particular modules.

Please put your thoughts on this.